src/inapp/types.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-use-before-define */
```

Comment:
Some types have just been reordered bc it's an easy resolution of this lint error